deployment/terraform/info.go

```diff
@@ -54,7 +54,10 @@ func displayInfo(output *Output) {
 	}
 
 	if output.HasProxy() {
-		fmt.Println("Proxy: " + output.Proxy.PublicIP)
+		fmt.Println("Proxy(ies):")
```

Comment:
Can we add the humblest `if` here to choose between "Proxy" and "Proxies"? Please :pleading_face:  :pray:

Reply:
:joy: :joy: 

Of course